src/runtime/cssFunctions.ts

```diff
@@ -1,10 +1,12 @@
-import {TinyColor} from '@ctrl/tinycolor';
+import {TinyColor, isReadable} from '@ctrl/tinycolor';
```

Comment:
can you just make sure it won't add tons of KB to all the bundles? i assume it's a small function but let's verify

Reply:
@ronnyrin any suggestions on how I can do that 🤔 ?

Reply:
you can build it, copy the dist to one of the artifacts that use it and compare the bundle size with webpack analyzer

Reply:
I think it would be a good idea for the future to integrate prefer

Reply:
Yeah, my question was totally a harama le hanchata for perfer 😄

Reply:
@ronnyrin do you have an artifact I can test this with 😬  🙏 ?

Reply:
https://github.com/wix-private/ecom/tree/master/client/wixstores-client/wixstores-client-cart-icon
we can do it together tomorrow if you don't want to clone it and stuff

Reply:
Np, I can try.
Thanks!

Reply:
![Screen Shot 2020-11-19 at 15 52 32](https://user-images.githubusercontent.com/3277080/99675095-76f65380-2a7f-11eb-87f9-7e2e099cfa61.png)
![Screen Shot 2020-11-19 at 15 51 15](https://user-images.githubusercontent.com/3277080/99675105-79f14400-2a7f-11eb-8554-1e44d92d3021.png)
Gzipped difference of 0.24KB

Reply:
@ronnyrin what do you say?